broker/src/main/java/org/apache/rocketmq/broker/controller/ReplicasManager.java

```diff
@@ -600,6 +600,7 @@ private void confirmNowRegisteringState() {
         if (this.brokerMetadata.isLoaded()) {
             this.registerState = RegisterState.CREATE_METADATA_FILE_DONE;
             this.brokerControllerId = brokerMetadata.getBrokerId();
+            this.haService.setBrokerControllerId(this.brokerControllerId);
```

Comment:
If it is the first time to start (without brokerIdentity file), the brokerControllerId will be forgotten to be set to HAService.

Reply:
Yes, you are right.
I have modify the code to set brokerControllerId when it starts without brokerIdentity file.